piecewise_web/index.html

```diff
@@ -401,17 +406,28 @@ <h3 class="league-gothic">Thank you!</h3>
 map.scrollWheelZoom.disable();
 var control = L.control.zoom({position: 'topright'});
 map.addControl(control);
-var osmLayer = L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', {
-	attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a>' +
-		'contributors'
-});
 
+// Use Open Street Maps as a base tile layer
+// var osmLayer = L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', {
+//	attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a>' +
+//		'contributors'
+// });
+
+// Use Mapbox as a base tile layer
+// var mapboxLayer = L.tileLayer(
+//		'https://{s}.tiles.mapbox.com/v3/newamerica.lcl1jan5/{z}/{x}/{y}.png', {
+//	attribution: '&copy; <a href="http://mapbox.com/">Mapbox</a>'
+// });
+ 
+// Use Mapbox as a base tile layer
 var mapboxLayer = L.tileLayer(
 		'https://{s}.tiles.mapbox.com/v3/newamerica.lcl1jan5/{z}/{x}/{y}.png', {
 	attribution: '&copy; <a href="http://mapbox.com/">Mapbox</a>'
 });
 
-// Set the default base tile layer
+// Set the default base tile layer. 
+// If using Open Street Maps: map.addLayer(osmLayer);
```

Comment:
The intended room is that out of the box, a fork will be able to deploy the default Seattle example map, and by following the docs, one can customize for a new location. So the mapbox layer should be enabled by default, and the instructions discuss uncoommenting the osm layer code and commenting the mapbox lines if an osm base layer is desired.

I'll double check that the instructions in config.md cover uncoommenting both the layer assignment lines and the lines that add the osm layer to the map.